StudCity.Frontend/src/pages/Groups/index.jsx

```diff
@@ -1,19 +1,48 @@
 import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
 import Container from '../../components/Container';
 import SearchField from '../../UI/fields/SearchField';
-
+import SelectField from '../../UI/fields/SelectField';
+import Scroller from '../../components/Scroller';
+import { VIEW_MODE_ICONS, GROUP_MOCK } from '../../utils/constants';
+import { selectViewMode, changeViewMode } from '../../app/features/groupsSlice';
+import GroupsModes from './GroupsModes';
 const Groups = () => {
+	const options = ["Default", "Descending of popularity", "Ascending of popularity"];
```

Comment:
better practice to declare such constants outside React component, and all Uppercased

Reply:
@Vova-Romaniuk Don`t miss this comment